src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -1868,7 +2013,7 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
         });
     });
 
-    eval edge_refresh_on_node_hover ([nodes,edges,model]((_,target)) {
+    eval snap_source_to_node ([nodes,edges,model]((target)) {
```

Comment:
unnecesary parenthesis - this should be catched by linter btw

Reply:
Linter does not complain but it is fixed.